packages/pigeon/example/app/ios/Runner/Messages.g.swift

```diff
@@ -14,6 +14,32 @@ import Foundation
   #error("Unsupported platform.")
 #endif
 
+/// Error thrown by Pigeon. Encapsulates a code, message, and details.
+class PigeonError: Swift.Error {
+  let code: String
+  let message: String?
+  let details: Any?
+
+  init(code: String, message: String?, details: Any?) {
+    self.code = code
+    self.message = message
+    self.details = details
+  }
+
+  var localizedDescription: String {
+    let detailsDescription: String
+    if let convertibleObject = details as? CustomStringConvertible {
```

Comment:
does string interpolation already cover the `CustomStringConvertible` case?

Reply:
Yeah. I thought I had to check multiple cases, but aparently the simple solution is enough. Done in c2beea180e17fa6ed3b3400206c0d5ae98ebff75.